schemas/network.yml

```diff
@@ -27,6 +27,22 @@
         Host IP address when the source IP address is the proxy.
       example: 192.1.1.2
 
+    - name: session_id
+      type: keyword
+      description: >
+        This is the session ID or connection ID,
```

Comment:
Reading in this field and the next one about connection, I wonder if we should introduce `connection.* as mentioned in an other thread instead of uptting it under network.

Reply:
Sorry @ruflin I forgot to reply before closing this PR out for splitting.  Are you just talking about a name change i.e. s/network/connection?  The network.* field set is intended to pick up flow and connection-based fields, and also network events that are not flow/connection related. I think this requires a bit more thought before making a decision.

Reply:
I'm actually thinking if we could use a network and a connection prefix. I think a big chunk of the info we have right now in `network` is in the right place, but there a things like `forwarded_ip` which probably fit better into connection and also these fields here.

Definitively needs more discussions, just an idea.